src/main/java/core/screens/GameScreen.java

```diff
@@ -1,87 +1,62 @@
 package core.screens;
 
 import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.Input;
 import com.badlogic.gdx.Screen;
-import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.graphics.Color;
 import com.badlogic.gdx.graphics.GL20;
-import com.badlogic.gdx.graphics.OrthographicCamera;
-import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
 import com.badlogic.gdx.maps.tiled.TiledMapRenderer;
 import com.badlogic.gdx.maps.tiled.TmxMapLoader;
 import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
-import core.GdxGame;
-import core.characters.CharacterSprite;
+import core.camera.CameraManager;
 
 public class GameScreen implements Screen {
 
     private TiledMapRenderer renderer;
-    private OrthographicCamera camera;
-//    private CharacterSprite sprites;
-    private float x;
-    private float y;
+    private CameraManager cameraManager;
+    private ShapeRenderer shapeRenderer;
 
     public GameScreen() {}
```

Comment:
Maybe we should also initiate a new CameraManager in the constructor in case that the render method is called before the show method. Otherwise, it might result in an error.

Reply:
Oh, I realised that the show method is automatically called once this screen becomes the current screen. Thus, this issue will never occur.

Reply:
not sure if that actually happens, since calls to `.render` should all be done by LibGDX

I agree tho about our own classes tho, if they are at all accessible by our own classes